draft-js-mention-plugin/src/mentionStrategy.js

```diff
@@ -1,12 +1,12 @@
 import { Entity } from 'draft-js';
 
-const findMention = (character) => {
+const findMention = (trigger) => (character) => {
   const entityKey = character.getEntity();
-  return (entityKey !== null && Entity.get(entityKey).getType() === 'mention');
+  return (entityKey !== null && Entity.get(entityKey).getType() === `mention-${trigger}`);
```

Comment:
I will merge it and fix it, that in case of `@` we keep the default of `mention` type.